packages/client/e2e/credit.spec.ts

```diff
@@ -1,32 +1,36 @@
-import { fin } from "@openfin/core"
 import { BrowserContext, expect, Page } from "@playwright/test"
 
 import { test } from "./fixtures"
+import {
+  CreditBlotterPageObject,
+  CreditNewRfqPageObject,
+  CreditRfqTilesPageObject,
+  CreditSellSidePageObject,
+} from "./pages"
 import { ExpectTimeout, isOpenFin } from "./utils"
 
-test.describe("Credit", () => {
-  let browserContext: BrowserContext
-  let newRfqPage: Page
-  let rfqsPage: Page
-  let rfqBlotterPage: Page
+let browserContext: BrowserContext
+let mainPage: Page
+let newRfqPage: CreditNewRfqPageObject
+let rfqsPage: CreditRfqTilesPageObject
+let rfqBlotterPage: CreditBlotterPageObject
 
+test.describe("Credit", () => {
   test.beforeAll(async ({ creditPagesRec }, workerInfo) => {
-    newRfqPage = creditPagesRec["credit-new-rfq"]
-    rfqsPage = creditPagesRec["credit-rfqs"]
-    rfqBlotterPage = creditPagesRec["credit-blotter"]
+    mainPage = creditPagesRec.mainPage
+    newRfqPage = creditPagesRec.newRfqPO
+    rfqsPage = creditPagesRec.rfqsPO
+    rfqBlotterPage = creditPagesRec.blotterPO
 
     if (isOpenFin(workerInfo)) {
-      const mainWindow = creditPagesRec["mainWindow"]
-
-      await mainWindow.evaluate(async () => {
-        fin.Window.getCurrentSync().maximize()
+      await mainPage.evaluate(async () => {
```

Comment:
to be removed alongside work to sort out RFQ Tiles filter in responsive mode - SU-68